ReactAndroid/src/main/java/com/facebook/react/modules/accessibilityinfo/AccessibilityInfoModule.java

```diff
@@ -189,6 +196,41 @@ public void announceForAccessibility(String message) {
     mAccessibilityManager.sendAccessibilityEvent(event);
   }
 
+  @Override
+  public void getInstalledAccessibilityServiceList(Callback successCallback) {
+    List<AccessibilityServiceInfo> servicesList =
+        mAccessibilityManager.getInstalledAccessibilityServiceList();
+
+     WritableArray result =
+        Arguments.fromArray(
+            servicesList.stream()
+                .map(
+                    service -> {
+                      Bundle map = new Bundle();
+                      map.putString("id", service.getId());
+                      map.putString(
+                          "capabilities",
+                          AccessibilityServiceInfo.capabilityToString(service.getCapabilities()));
+                      map.putString(
+                          "eventTypes", AccessibilityEvent.eventTypeToString(service.eventTypes));
+                      map.putString(
+                          "feedbackType",
+                          AccessibilityServiceInfo.feedbackTypeToString(service.feedbackType));
+                      map.putString("flags", AccessibilityServiceInfo.flagToString(service.flags));
```

Comment:
Same issue as with capabilities above.  

This issue is why your example has "null" for flags for the Talkback service, which should have a number of flags set as seen here:

https://github.com/google/talkback/blob/92eb6dd4461e53fc904052b7fbe9b77ddfbf930a/talkback/src/main/java/TalkBackService.java#L1446-L1466